src/main/java/se/sundsvall/smloader/integration/db/model/CaseMapping.java

```diff
@@ -0,0 +1,139 @@
+package se.sundsvall.smloader.integration.db.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.IdClass;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.PreUpdate;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+import jakarta.validation.constraints.NotNull;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Objects;
+
+@Entity
+@IdClass(CaseMappingId.class)
+@Table(name = "case_mapping",
+	uniqueConstraints = {
+		@UniqueConstraint(name = "uq_external_case_id", columnNames = {"externalCaseId"})
+	})
+public class CaseMapping {
+
+	@Id
+	@Column(unique = true, name = "externalCaseId")
+	private String externalCaseId;
+
+	@Id
+	@Column(name = "caseId")
+	private String caseId;
+
+	@NotNull
+	@Column(name = "caseType", columnDefinition = "varchar(255)")
+	private String caseType;
+
+	@Column(name = "serviceName")
+	private String serviceName;
+
+	@Schema(accessMode = Schema.AccessMode.READ_ONLY)
+	@Column(name = "timestamp")
+	private LocalDateTime timestamp;
```

Comment:
Add `@TimeZoneStorage(TimeZoneStorageType.NORMALIZE)`?

Reply:
Should probably use OffsetDateTime aswell